docs/building.md

```diff
@@ -8,12 +8,12 @@ video drivers for your card.
 
 ### Windows
 
-* Windows 8 or 8.1
+* Windows 8 or later
 * Visual Studio 2015 or Visual Studio 2017
-* [Python 2.7](https://www.python.org/downloads/release/python-2713/)
+* [Python 3.6](https://www.python.org/downloads/release/python-362/)
```

Comment:
@DrChat Python 3 is ready to be pulled I think. #755 pull request which I think needs 3.4+. I will change it than if need be.